core/station/impl/src/core/init.rs

```diff
@@ -166,17 +165,6 @@ lazy_static! {
             Allow::user_groups(vec![*ADMIN_GROUP_ID]),
             Resource::ExternalCanister(ExternalCanisterResourceAction::Fund(ExternalCanisterId::Any)),
         ),
-        (
-            Allow::user_groups(vec![*ADMIN_GROUP_ID]),
-            Resource::ExternalCanister(ExternalCanisterResourceAction::Call(CallExternalCanisterResourceTarget {
-              validation_method: ValidationMethodResourceTarget::No,
-              execution_method: ExecutionMethodResourceTarget::Any,
-            })),
-        ),
-        (
-            Allow::user_groups(vec![*ADMIN_GROUP_ID]),
-            Resource::ExternalCanister(ExternalCanisterResourceAction::Read(ExternalCanisterId::Any)),
```

Comment:
this should not be removed, we should keep the same permissions as we have now on main for the external canisters, might be worth it to make sure this branch is up to date with latest main.

Reply:
Good catch, I mused have messed this up when merging main

Reply:
Ah I see now I removed this because it was a duplicate, the same policy was already in the list. Probably due to a merge error from before.